bin/alluxio-masters.sh

```diff
@@ -37,8 +37,21 @@ ALLUXIO_TASK_LOG="${ALLUXIO_LOGS_DIR}/task.log"
 
 echo "Executing the following command on all master nodes and logging to ${ALLUXIO_TASK_LOG}: $@" | tee -a ${ALLUXIO_TASK_LOG}
 
+check_true() {
+    local output=$1
+    if [[ $output == *"true"* ]]; then
+        result="true"
+    else
+        result="false"
+    fi
+    echo $result
```

Comment:
remove echo

Reply:
Actually I don't think it can be removed. If we delete echo, the function will use a temporary parameter "result" to store the boolean value, after returning, the result will not exist.